build/serve.js

```diff
@@ -102,7 +102,7 @@ gulp.task('serve:watch', ['watch'], serveDevelopmentMode);
 /**
```

Comment:
The reason that spawn-backend uses binary from .tmp is that the .tmp binary compiles much faster than dist one. Compilation options are different. What's more, the dist binary runs only on linux. This is because everything in dist is going to be packaged into a docker image. Do you have any idea on how to better arrange this?